packages/typespec-azure-vscode/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "typespec-azure-vscode",
+  "displayName": "typespec azure",
+  "description": "the extension for azure typespec",
+  "publisher": "typespec",
+  "version": "0.1.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Azure/typespec-azure.git"
+  },
+  "engines": {
+    "vscode": "^1.100.0"
+  },
+  "extensionDependencies": [
+    "typespec.typespec-vscode"
+  ],
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [
+    "onLanguage:typespec",
+    "workspaceContains:**/tspconfig.yaml"
+  ],
+  "main": "./dist/src/extension.cjs",
+  "preview": true,
+  "contributes": {
+    "languages": [
+      {
+        "id": "typespec",
+        "aliases": [
+          "TypeSpec",
+          "TYPESPEC"
+        ],
+        "extensions": [
+          ".tsp"
+        ],
+        "icon": {
+          "light": "./icons/tsp-file.light.svg",
+          "dark": "./icons/tsp-file.dark.svg"
+        },
+        "configuration": "./dist/language-configuration.json"
+      },
+      {
+        "id": "markdown-typespec"
+      }
+    ]
+  },
+  "scripts": {
+    "vscode:prepublish": "npm run compile",
+    "compile": "npm run check-types && tsx ./scripts/build.ts",
```

Comment:
don;'t use `npm run` replace with `pnpm` this breaks usage without npm and logs warnings with newer npms